cmake_modules/SnappyConfig.h

```diff
@@ -0,0 +1,8 @@
+#ifndef SNAPPY_CONFIG_H
```

Comment:
Similar like above, I've decided to use optional config.h to define missed C++ language type to be included in snappy build